src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -149,7 +149,7 @@ const Index = ({
                 <Paper className={classes.paper}>
                     <div className={classes.maxWidth}>
                         <div className={classes.title} >
-                            Search for {titleText}
+                            {i18n.t('Search for ')} {titleText}
```

Comment:
I'm sorry for mentioning this in a back port, but we generally try to avoid splitting translatable strings into different parts. The reason being that the sentence structure could be different in an another language (I'm definitely no language expert, but I think this link highlights the idea: https://www.babbel.com/en/magazine/foreign-language-sentence-structure/).

Maybe you could add a second param to the `useScopeTitleText` declaration and concatenate this with the rest in the i18n function? 

There are a couple of more places where `useScopeTitleText` is used, so probably need to updated those too 🙂 

If this doesn't make sense, please reach out and we can discuss.